alacritty_terminal/src/tty/unix.rs

```diff
@@ -143,7 +144,7 @@ fn default_shell(pw: &Passwd<'_>) -> Program {
 }
 
 /// Create a new TTY and return a handle to interact with it.
-pub fn new<C>(config: &Config<C>, size: &SizeInfo, window_id: Option<usize>) -> Pty {
+pub fn new(config: &PtyConfig, size: &SizeInfo, window_id: Option<usize>) -> Result<Pty> {
```

Comment:
It's required to not crash entire process when you do something like `alacritty msg create-window -e not_found`, since without it you'd bring entire process down instead of just a failure to create window.

Reply:
Makes sense anyway, better do it correct right away. Great.